cv32/tests/programs/custom/hpmcounter_basic_test/hpmcounter_basic_test.c

```diff
@@ -53,6 +53,17 @@ static int chck(unsigned int is, unsigned int should)
   return err;
 }
 
+static int chck_le(unsigned int is, unsigned int should)
+{
+  int err;
+  err = is <= should ? 0 : 1;
```

Comment:
Ha!   I have not seen "is" and "should" used in place of "actual" and "expected".  :-)